src/utils/transform-article-strapi-data.js

```diff
@@ -47,6 +47,7 @@ export const transformArticleStrapiData = article => {
                 url: SEOObject.og_url,
             },
             twitter: {
+                card: SEOObject.twitter_card,
```

Comment:
if twitter card is not set in CMS, will it set summary as as twitter card?

Reply:
yes, we updated the logic here as well in SEO.js to default to summary